code/datums/gamemode/factions/bloodcult/bloodcult_effects.dm

```diff
@@ -420,3 +420,108 @@ var/list/bloodstone_backup = 0
 					)
 		new mobtype(get_turf(src))
 		qdel(src)
+
+///////////////////////////////////////STUN INDICATOR////////////////////////////////////////////////
+/obj/effect/stun_indicator
+	icon = null
+	anchored = 1
+	mouse_opacity = 0
+	var/list/viewers = list()
+	var/image/indicator = null
+	var/current_dots = 6
+	var/mob/victim = null
+
+/obj/effect/stun_indicator/New()
+	..()
+	if (!ismob(loc))
+		qdel()
+		return
+
+	victim = loc
+	current_dots = Clamp(round(victim.knockdown/2.5),0,5)
+
+	if (!current_dots)
+		qdel()
+		return
+
+	current_dots++//so we get integers from 1 to 6
+
+	for (var/mob/M in player_list)
+		if (iscultist(M) && M.client)
+			viewers += M.client
+
+	indicator = image(icon='icons/obj/cult.dmi',loc=victim,icon_state="",layer=SNOW_OVERLAY_LAYER)
+	update_indicator()
+
+/obj/effect/stun_indicator/proc/update_indicator()
+	spawn()
+		while (victim && (victim.stat < 2) && victim.knockdown)
```

Comment:
Ew. Spawn isn't some magic desynchroniser, it just puts it to the back of the work queue.

Set waitfor = FALSE should be better, and even then this is not very good.

Reply:
Well I just want it executed after the talisman's been deleted, because otherwise you've got infinite-use stun talismans. will waitfor = FALSE do that?

Reply:
There's also the magic `2`